_includes/headtags.html

```diff
@@ -1,14 +1,13 @@
 <meta charset="utf-8">
 <!-- Canonical and Alternate Links -->
+<link rel="canonical" hreflang="en" href="https://jamulus.io{{ page.url }}">
 {% assign sorted_languages = site.languages | sort %}
 {% for lng in sorted_languages %}
-  {% if lng == site.default_lang %}
-    <link rel="canonical" href="{{ site.url }}{{ site.baseurl }}{{ page.url }}">
+  {% if lng == site.active_lang %}
```

Comment:
Change the empty `if` block using `==` to use `!=` (so making the `else` content the `if` content and removing the `else`).